tr/advanced/best-practice-security.md

```diff
@@ -1,69 +1,69 @@
+_"canlı ortam"_ terimi, bir uygulama veya API'nin genel olarak son kullanıcıları veya tüketicileri için hazır olduğu yazılım yaşam döngüsündeki aşamayı ifade eder. Buna kıyasla, _"geliştirme"_ aşamasında aktif olarak kod yazıyor ve test ediyorsunuz, ve uygulama dış erişime açık değildir. Bunlara karşılık gelen sistem ortamları sırasıyla _canlı ortam (production)_ ve _geliştirme (development)_ ortamları olarak bilinir.
 
-{% include note.html content="If you believe you have discovered a security vulnerability in Express, please see
-[Security Policies and Procedures](https://github.com/expressjs/express/blob/master/Security.md).
-" %}
+Canlı ve geliştirme ortamları genel olarak farklı şekilde kurulurlar ve çok farklı gereksinimleri vardır. Geliştirme ortamında iyi olan bir şey canlı ortamda kabul edilebilir olmayabilir. Örneğin, geliştirme ortamında hata ayıklama için ayrıntılı hataların loglanmasını isteyebilirsiniz, ancak aynı şey canlı ortamda güvenlik açığı oluşturabilir. Ve geliştirme ortamında ölçeklenebilirlik, güvenilirlik ve performans hakkında endişe etmenize gerek yok iken, bu konular canlı ortamda kritikleşir.
 
-Security best practices for Express applications in production include:
+{% include note.html content="Express'te bir güvenlik açığı keşfettiğinizi düşünüyorsanız, lütfen bakınız
+[Güvenlik Politikaları ve Prosedürleri](/en/resources/contributing.html#security-policies-and-procedures).
+" %}
 
-- [Don’t use deprecated or vulnerable versions of Express](#dont-use-deprecated-or-vulnerable-versions-of-express)
-- [Use TLS](#use-tls)
-- [Use Helmet](#use-helmet)
-- [Use cookies securely](#use-cookies-securely)
-- [Ensure your dependencies are secure](#ensure-your-dependencies-are-secure)
-- [Avoid other known vulnerabilities](#avoid-other-known-vulnerabilities)
-- [Additional considerations](#additional-considerations)
+Canlı ortamdaki Express uygulamaları için en iyi güvenlik pratikleri:
 
+- [Express'in kullanımdan kaldırılmış veya bakımı yapılmayan versiyonlarını kullanmayın](#dont-use-deprecated-or-vulnerable-versions-of-express)
+- [TLS kullanın](#use-tls)
+- [Helmet kullanın](#use-helmet)
+- [Çerezleri güvenli kullanın](#use-cookies-securely)
+- [Otorizasyona karşı yapılan brute-force saldırılarını engelleyin](#prevent-brute-force-attacks-against-authorization)
+- [Bağımlılıklarınızın güvende olduğundan emin olun](#ensure-your-dependencies-are-secure)
+- [Bilinen diğer güvenlik açıklarından kaçının](#avoid-other-known-vulnerabilities)
+- [Ek hususlar](#additional-considerations)
```

Comment:
Links to anchors should be also updated to translated version. Otherwise this list won't "jump" to relevant section